spec/features/bulk_updates/bulk_updates_location_keywords_spec.rb

```diff
@@ -38,11 +36,15 @@
     it 'displays the preview information', bulk_update_step_1: true do
       expect(page).to have_content('Preview of New MMT_2 Bulk Update')
 
-      expect(page).to have_content("Name #{bulk_update_name}")
-      expect(page).to have_content('Field to Update Location Keywords')
-      expect(page).to have_content('Update Type Find And Remove')
+      expect(page).to have_content('Name')
+      expect(page).to have_content(bulk_update_name)
+      expect(page).to have_content('Field to Update')
+      expect(page).to have_content('Location Keywords')
+      expect(page).to have_content('Update Type')
+      expect(page).to have_content('Find And Remove')
+
       within '.find-values-preview' do
-        expect(page).to have_content('CATEGORY: ANY VALUEARCTIC')
+        expect(page).to have_content('CATEGORY: ANY VALUE ARCTIC', normalize_ws: true)
```

Comment:
https://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Matchers#has_text%3F-instance_method, we are using the `:normalize_ws` option, which says 'When true replace all whitespace with standard spaces and collapse consecutive whitespace to a single space'.

Again, there is a difference of how science and location keywords get rendered between the previous webdriver/headless browser and the new one. Now the spaces are there (as you can see previously the space was not there). The styling for these keywords has not changed and still looks good, I'm just adding the normalize_ws to make sure all the different whitespace characters get treated the same because it will all try to render on one line.

Reply:
yea, there seams to be a different opinion between these tools on what to do with white space. I like the new code better.

Reply:
right but in this case, Poltergeist/PhantomJS actually didn't even show a space in between the two values.